CIP-0072/README.md

```diff
@@ -51,7 +52,7 @@ Stores and auditors should be able to follow the chain and find when a new dApp
 
 ### **On-chain dApp Registration Certificate**
 
-The on-chain dApp registration certificate MUST follow canonical JSON and be serialised according to RFC 8785 (https://www.rfc-editor.org/rfc/rfc8785). This stipulation is to avoid any ambigiutines in the signature calculation.
+The on-chain dApp registration certificate MUST follow canonical JSON and be serialised according to RFC 8785 (https://www.rfc-editor.org/rfc/rfc8785).
```

Comment:
Since we removed the signature, we don't need to be `canonical` anymore

Reply:
removed 🙂. Thank you!